### ISSUE TITLE
How to directly use word embedding from the pre-trained LM during training and inference?

### ISSUE THREAD

**Issue by Fishersponge** [Pay20Y/SEED#7]
If I have a 'train.mdb', how can I use the fasttext pre-train model **cc.en.300.bin**? I see **nothing about fasttext** in your **./models and trainers.py and main.py.** Waiting for your answer, thanks~

**Comment by Pay20Y** [comment]
Hi, please refer to create_all_synth_lmdb.py and modify the dataloader accordingly.

**Comment by Ma01180724** [comment]
hello，i have a same question，we have the datasets nips2014 and cvpr 2016 （lmdb），how can i use the fasttext pre-train model？can you solve it？ thanks

**Comment by Ma01180724** [comment]
@Pay20Y ，could you give me datasets that you had prepared？

**Comment by Pay20Y** [comment]
@Ma01180724 Hi, I'm really sorry that I can't directly share the training datasets with you because of the large storage. There are two ways to get the training datasets by yourself. First, You can modify the create_all_synth_lmdb.py that load the labels from MJ and ST then generates the new LMDB datasets with embedding labels. Second, as mentioned before, you can modify the dataloader and generate the according word embedding from the recognition label during the training process.
